src/irc/BridgedClient.ts

```diff
@@ -793,14 +795,7 @@ export class BridgedClient extends EventEmitter {
             if (reason === "banned") {
                 // If we've been banned, this is intentional.
                 this._explicitDisconnect = true;
-            }
-
-            // make sure a killed connection stays killed
-            if (this.state.status !== BridgedClientStatus.KILLED) {
-                this._state = {
-                    status: BridgedClientStatus.DEAD,
```

Comment:
I am not sure why you removed this, i think now the status will stay on `CONNECTED` if the bridge get disconnected from irc because of some network error.

Reply:
Because I read `this.state.status === BridgedClientStatus.KILLED` :facepalm:. Now it makes sense